lib/index.ts

```diff
@@ -609,6 +618,96 @@ export class Worker {
 				...CONTRACTS,
 				...self.getTypeContracts(),
 			},
+			executeSubsctiptionsLib: (
```

Comment:
Since these submodules are only used inside the `action-evaluate-trigger` module, I think it would be sensible to import them directly there, instead of making them part of the generic interface.

Reply:
Oh that's nicer. In a way, this would be like making them `protected` instead of the `public` WorkerContext that is available to other actions.

Reply:
Tried again to refactor this, and remembered that the issue is that the action doesn't have a reference to the worker - only to the WorkerContext, and the three functions ( `executeTriggers`, `executeSubscriptionsLib`, and `executeEvaluateTransformers`) are _methods_ of the Worker. The functions use several properties of the Worker that are not part of the WorkerContext, so passing a function in the WorkerContext is the cleanest I could find.

I could clean it up a bit by passing just one function instead of the three previous ones, but nevertheless, it's true that the `action-evaluate-triggers` is the only action that would use it.

Reply:
Cleaned it up by using just one function on the `WorkerContext`.